server/src/main/java/org/elasticsearch/index/query/functionscore/FieldValueFactorFunctionBuilder.java

```diff
@@ -144,7 +144,7 @@ protected int doHashCode() {
 
     @Override
     protected ScoreFunction doToFunction(QueryShardContext context) {
-        MappedFieldType fieldType = context.getMapperService().fieldType(field);
+        MappedFieldType fieldType = context.fieldMapper(field);
```

Comment:
Maybe we should rename this `fieldType` rather than `fieldMapper`?

Reply:
yes indeed, I planned to do it as a follow-up